data/remnant missions.txt

```diff
@@ -493,5 +592,771 @@ mission "Remnant: Technology Available"
+			choice
+				`	"Yes, you are welcome to examine it."`
+					goto deep
+				`	"No, stay away from my ship."`
+					decline
+				`	"Yes, but don't touch anything."`
+
+			`	"Don't worry," he sings with a tinge of reproach. "I am a Remnant, not a Korath."`
+				goto more
+
+			label deep
+			`	He clips his harness to one of the overhead lifts and presses a button on the back of his gauntlet, and is promptly whisked up into the air. The lift appears designed to give engineers and techs almost as much freedom as a zero-g environment by suspending them with just enough force to counteract gravity, so they can move around a ship with ease. After a quick once-over he quickly gravitates to the weapon ports and turret hardpoints.`
+
+			label more
+			`	After examining <ship> for quite some time, he calls down to you. "I have never seen weapons designed to handle such a high heat output! Is this some kind of plasma based weapon?" he asks, gesturing at a Plasma Cannon. You nod, and he continues, "Intriguing. We have noticed that the Korath ships seem to run particularly hot, and have speculated that overheating them might be an effective means of disabling them. I would enjoy researching these a bit more. If you could acquire some for us you would be well compensated."`
+			choice
+				`	"Sure, I would be glad to work out a deal."`
+				`	"Sorry, I'm really not interested in helping Remnant research."`
+					decline
+
+			`	After a bit of haggling, he agrees to pay you one million credits in exchange for a cargo of two "Plasma Cannons."`
+				accept
+	
+	on complete
+		outfit "Plasma Cannon" -2
+		payment 1500000
+		conversation
+			`As <ship> settles onto the pad closest to the research facility, you can see the weapons specialist checking over a large cannon on a platform beneath a nearby Albatross. When he sees you starting to unload the two plasma cannons he snaps the casing closed and gestures to someone out of sight. Within moments both him and an empty flatbed are there to pick up the weapons.`
+			`	"Thank you for your help," he says. "Our Inhibitors provide us with something of a mobility advantage against the Korath, but they aren't particularly effective at disabling them. These plasma cannons will open new avenues of research for us." He pauses, then continues. "I also included a bonus for you. It is a 'finder's fee,' as we always appreciate new tech."`
+			`	The weapon tech finishes strapping down the plasma cannons, and looks up to see another Remnant approaching. He gives the new Remnant a respectful dip of his head that almost seems like a bow before turning to you and chanting, "Ah, Captain <last>, have you met Tali? She is the engineering prefect here on Viminal. She asked me to let her know when or if you managed to procure the new tech for me."`
```

Comment:
Capitalize again.
Tech -> technician. Drop the comma before `and` in the first sentence.
Again with the lack of branches for the meeting of Tali.